csharp/examples/IoTExample/IoTExample.csproj

```
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

```suggestion
    <TargetFramework>netcoreapp3.1</TargetFramework>
```

2.1 is EOL'd. Can you update this to 3.1?